config/initializers/extra_paths.rb

```diff
@@ -0,0 +1,6 @@
+#
+# Add some extra paths here
+Rails.application.configure do
+  config.paths.add File.join('app', 'api'), glob: File.join('**', '*.rb')
+  config.autoload_paths += Dir[Rails.root.join('app', 'api', '*')]
```

Comment:
If i don't have mistake,  app modules automatic load. For example, if you add validator's folder - you don't need additional settings. 
Need trying